src/bin/cql-stress-cassandra-stress/java_generate/values/varint.rs

```diff
@@ -0,0 +1,29 @@
+use scylla::frame::{response::result::CqlValue, value::CqlVarint};
+
+use crate::java_generate::distribution::Distribution;
+
+use super::{ValueGenerator, ValueGeneratorFactory};
+
+#[derive(Default)]
+pub struct VarInt;
+
+impl ValueGenerator for VarInt {
+    fn generate(
+        &mut self,
+        identity_distribution: &mut dyn Distribution,
+        _size_distribution: &mut dyn Distribution,
+    ) -> CqlValue {
+        // BigInt::from_i64 returns Some(). This unwrap is safe.
+        CqlValue::Varint(CqlVarint::from_signed_bytes_be(
+            identity_distribution.next_i64().to_be_bytes().to_vec(),
+        ))
+    }
```

Comment:
This is allegedly equivalent code in Java:
```java
    @Override
    public BigInteger generate()
    {
        return BigInteger.valueOf(identityDistribution.next());
    }
```
Can you prove that it's indeed equivalent? Tests are also needed.